cloudfoundry-client-spring/src/test/java/org/cloudfoundry/spring/client/v2/routemappings/SpringRouteMappingsTest.java

```diff
@@ -79,4 +84,38 @@ protected CreateRouteMappingRequest getValidRequest() throws Exception {
         }
     }
 
+    public static final class Delete extends AbstractApiTest<DeleteRouteMappingRequest, DeleteRouteMappingResponse> {
```

Comment:
This is missing a test for `DeleteAsync`.  Since the payload of async operations is different (the `JobEntity` is returned for polling) we maintain a separate test for that as well.  I'll add it in a follow up commit.